lib/src/main/java/org/team100/lib/localization/NotePositionStruct.java

```diff
@@ -0,0 +1,41 @@
+package org.team100.lib.localization;
+
+import java.nio.ByteBuffer;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.util.struct.Struct;
+
+public class NotePositionStruct implements Struct<NotePosition> {
+
+    @Override
+    public Class<NotePosition> getTypeClass() {
+        return NotePosition.class;
+    }
+
+    @Override
+    public String getTypeString() {
+        return "struct:NotePosition";
+    }
+
+    @Override
+    public int getSize() {
+        return kSizeInt32 + Translation2d.struct.getSize();
+    }
+
+    @Override
+    public String getSchema() {
+        return "int id;Translation2d pose";
```

Comment:
the "schema" here is supposed to reflect the contents of the NotePosition object.  does this look right?